_posts/2025-01-08-working-with-graph-databases-in-python.md

```diff
@@ -0,0 +1,278 @@
+---
+title:  "Working with RDF Graph Databases in Python"
+author: Leo Rudczenko
+categories:
+  - Software Engineering
+tags:
+  - python
+  - graph
+  - database
+  - rdf
+  - linked-data
+---
+
+## What are Graph Databases?
+
+The logic behind a [Graph Database](https://en.wikipedia.org/wiki/Graph_database),
+comes from graph theory in mathematics, where a network of nodes representing data are connected using edges.
+This makes a graph database more flexible than a typical hierarchical database,
```

Comment:
suggest just simplify these paragraphs - lines 18 to 27 - to "This makes a graph database more flexible than a relational database, because it removes the need to pre-determine the types of entities and relationships that it can store".